CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Guidelines for Contributing to Dafny
+
+Thanks for contributing to Dafny!  Github is the right place to discuss feature requests, report issues with Dafny, or contact the Dafny developers.
+Dafny is an Open Source project and welcomes contributions.
+
+## Reporting issues
+
+Before reporting an issue here, consider whether it would be better handled in one of the following places:
+
+- [Stack Overflow](https://stackoverflow.com/questions/tagged/dafny), an online Q&A website, for help with writing and proving Dafny programs.
+- [`dafny-lang/ide-vscode`](https://github.com/dafny-lang/ide-vscode), Dafny's VSCode plugin, for issues or feature requests specific to the plugin itself and issues with Dafny's LSP server.
+- [`boogie-org/boogie-friends`](https://github.com/boogie-org/boogie-friends/), Dafny's Emacs mode, for Emacs-specific problems.
+- [`boogie-org/boogie`](https://github.com/boogie-org/boogie), Dafny's intermediate verification language, for questions and issues specific to Boogie.
+
+Other issues can be reported to the [Dafny project](https://githubcom/dafny-lang/dafny/Issues).
+Please include your Dafny version number, any relevant code, and expected results in the issue report.
+
+## Contributing to the source code
+
+Developers who would like to experiment with and perhaps modify Dafny source should note these resources:
+- First, there are [resources to learn about Dafny as a user](https://dafny.org/dafny/).
+- And note the [community Code of Conduct](https://github.com/dafny-lang/dafny/blob/master/.github/CODE_OF_CONDUCT.md) expected of all participating in the project.
+- The Dafny project is at [github](https://github.com/dafny-lang/dafny) with related [sibling repositories](https://github.com/dafny-lang).
```

Comment:
Right, but how much does it differ from the exit code returned by the main method?

Reply:
OK...